mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -41,65 +43,183 @@ description: Vertex AI Feature Online Store provides a centralized repository fo
 autogen_async: false
 examples:
   - !ruby/object:Provider::Terraform::Examples
-    name: vertex_ai_feature_online_store
-    primary_resource_id: feature_online_store
+    name: 'vertex_ai_featureonlinestore_with_bigtable'
+    primary_resource_id: 'featureonlinestore'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
     vars:
-      name: example_feature_online_store
+      name: 'terraform2'
+      project: 'appeng-flex'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    ignore_read_extra:
+      - 'force_destroy'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'vertex_ai_featureonlinestore_with_beta_fields_optimized'
+    primary_resource_id: 'featureonlinestore'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      name: 'terraform'
+      project: 'appeng-flex'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    ignore_read_extra:
+      - 'force_destroy'
+    min_version: beta
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'vertex_ai_featureonlinestore_with_beta_fields_bigtable'
+    primary_resource_id: 'featureonlinestore'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      name: 'terraform3'
+      project: 'appeng-flex'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    ignore_read_extra:
+      - 'force_destroy'
+    min_version: beta
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  pre_delete: templates/terraform/pre_delete/vertex_ai_force_delete.go.erb
+virtual_fields:
+  - !ruby/object:Api::Type::Boolean
+    name: 'force_destroy'
+    description:
+      'If set to true, any FeatureViews and Features for this FeatureOnlineStore will also be deleted.'
+    default_value: false
 parameters:
   - !ruby/object:Api::Type::String
     name: region
     description: The region of feature online store. eg us-central1
     url_param_only: true
     immutable: true
-    required: true
+    default_from_api: true
```

Comment:
Could you explain this change? This seems incorrect based on the url structure.

Reply:
I've updated it to use default_from_api : true because if region isn't provided, I wanted to use the default region set on the project.

Reply:
Ah right, thank you. This was incorrect in the previous PR then, but luckily it is not a breaking change.